src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -3462,7 +3476,10 @@ void SimAirLoopComponent(EnergyPlusData &state,
                          bool const FirstHVACIteration, // TRUE if first full HVAC iteration in an HVAC timestep
                          int const AirLoopNum,          // Primary air loop number
                          int &CompIndex,                // numeric pointer for CompType/CompName -- passed back from other routines
-                         HVACSystemData *CompPointer    // equipment actual pointer
+                         HVACSystemData *CompPointer,   // equipment actual pointer
+                         int const &airLoopNum,         // index to AirloopHVAC
+                         int const &branchNum,          // index to AirloopHVAC branch
+                         int const &compNum             // index to AirloopHVAC branch component
 )
```

Comment:
Adds three new arguments to SimAirLoopComponent() function that allows getting the pointer to the component simulated if it is not already set in the header file.